docs/docs/api/LayoutAnimations/AnimatedLayout.md

```diff
@@ -0,0 +1,71 @@
+---
+id: animatedLayout
+title: <AnimatedLayout>
+sidebar_label: <AnimatedLayout>
+---
+The component is responsible for observing changes in its subtree. There is no need to nest one `AnimatedLayout` in another one because the higher one would already watch the subtree of the lower one. You can treat it as a `View` component. `AnimatedLayout` manages all its descendants' animations while it's mounting and unmounting as well. 
```

Comment:
"The `AnimatedLayout` component ..."

Reply:
I think it is important to mention that the component should be placed somewhere in the tree in order for enter/exit/layout animations to work. You go straight to explaining that no nesting is needed w/o explaining why it is needed at all